pkg/serviceaccounts/controllers/docker_registry_service.go

```diff
@@ -18,6 +18,7 @@ import (
 	"k8s.io/kubernetes/pkg/watch"
 )
 
+// TODO: use clusterDomain instead of cluster.local
 const DefaultOpenshiftDockerURL = "docker-registry.default.svc.cluster.local:5000"
```

Comment:
I *think* this could just be changed to `docker-registry.default.svc:5000`... that should resolve just as well, and this is really a placeholder until the actual service IP of the registry is resolved. @deads2k @smarterclayton can you think of a reason that wouldn't work?